report/4.Solution/Lexer.tex

```diff
@@ -1,29 +1,29 @@
 %This section describes the Lexer
 %I have used lexer, tho I have described that a scanner is also valid wording
 %I possibly might need some reference to the EBNF section
+\section{Lexical Analysis}
 The first part of the compiler is concerned with the syntax analysis phase.
 This portion of the compiler is often called a lexer or a scanner.
 The point of the lexer is to transform a given input character stream, often a source code file, into a stream of tokens.
 
-The scanner will do this by breaking the character stream down into individual words, known in language theory as a lexeme.
+The scanner does this by breaking the character stream down into individual words, known in language theory as a lexeme.
```

Comment:
Nogle steder (som her) kalder vi det "scanner", andre steder (som i lige 7) kalder vi det "lexer".

Vi bør nok kun bruge det ene - jeg stemmer for "lexer", så det stemmer overens med vores kode.